ground/gcs/src/plugins/config/configinputwidget.cpp

```diff
@@ -1784,3 +1774,64 @@ void ConfigInputWidget::checkHangtimeConfig()
 
     m_config->lblHangTimeWarning->setVisible(warn);
 }
+
+void ConfigInputWidget::clearMessages(QWidget *widget, const QString type)
```

Comment:
Should probs move these message functions to ConfigTaskWidget so they can be used elsewhere.